chaincfg/params.go

```diff
@@ -69,13 +68,13 @@ type ConsensusDeployment struct {
 	// this particular soft-fork deployment refers to.
 	BitNumber uint8
 
-	// StartTime is the median block time after which voting on the
-	// deployment starts.
-	StartTime uint64
+	// DeploymentStarter is used to determine if the given
+	// ConsensusDeployment has started or not.
+	DeploymentStarter ConsensusDeploymentStarter
 
-	// ExpireTime is the median block time after which the attempted
-	// deployment expires.
-	ExpireTime uint64
+	// DeploymentEnder is used to determine if the given
+	// ConsensusDeployment has ended or not.
+	DeploymentEnder ConsensusDeploymentEnder
```

Comment:
This change also needs to be propagated to the RPC server (build errors below). We'll want to still expose `StartTime` and `EndTime` to populate the RPC response.

```
./rpcserver.go:1292:39: deploymentDetails.StartTime undefined (type chaincfg.ConsensusDeployment has no field or method StartTime)
./rpcserver.go:1293:39: deploymentDetails.ExpireTime undefined (type chaincfg.ConsensusDeployment has no field or method ExpireTime)
```